pkg/config/resolver.go

```diff
@@ -250,6 +254,46 @@ func (r *Resolver) UIClient() target.Client {
 	return r.uiClient
 }
 
+func (r *Resolver) YandexClient() target.Client {
```

Comment:
You need one configuration that has to be set to enable the output and early returns with null if not set. Like the endpoint for Loki or webhook for discord. Otherwise people get your notice and warning logs without want to use Yandex.